### PR TITLE
Add feature #825 - [Excel to PDF] Show a list of excel files in botto…

### PR DESCRIPTION
**Pull request by wseemann**
…m sheet

# Description

[Excel to PDF] Show a list of excel files in bottom sheet #825

Screen Shot 2019-10-22 at 1 26 50 AM

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@wseemann Are all these files password protected?
I can see "lock" icon in front of these - it means file is password protected

File: app/src/main/java/swati4star/createpdf/util/BottomSheetUtils.java [hunk 1]
```diff
@@ -25,4 +25,8 @@ public void populateBottomSheetWithPDFs(BottomSheetPopulate listener) {
         new PopulateBottomSheetList(listener, new DirectoryUtils(mContext)).execute();
     }
 
+    public void populateBottomSheetWithExcelFiles(BottomSheetPopulate listener) {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/BottomSheetUtils.java]: add javadoc style comments to this function

**Comment by wseemann** [comment]
> @wseemann Are all these files password protected?
> I can see "lock" icon in front of these - it means file is password protected

Yes, they are password protected.

**Comment by Swati4star** [comment]
@wseemann I just merged someone's PR and it is causing some merge conflicts. Please resolve those too

**Comment by Swati4star** [comment]
Updates @wseemann ?

**Pull request merged**